projects/fsastorefrontlib/src/occ/adapters/consent-management/occ-consent.adapter.spec.ts

```diff
@@ -8,15 +8,22 @@ import {
   HttpTestingController,
 } from '@angular/common/http/testing';
 import { TestBed, waitForAsync } from '@angular/core/testing';
-import { OccEndpointsService } from '@spartacus/core';
+import { Address, OccEndpointsService } from '@spartacus/core';
 import { OccConsentAdapter } from './occ-consent.adapter';
 
 const userId = 'testId';
 const cartId = 'cartId';
 const oboCustomerId = 'oboCustomerId';
+
 const consentsEndpoint = 'oboConsents';
 const oboConsentCustomersEndpoint = 'oboConsentCustomers';
 const transferCartEndpoint = 'transferCart';
+const addAddressEndpoint = 'oboConsentAddresses';
+
+const address: Address = {
+  companyName: 'msg Global Digital',
```

Comment:
Not sure if we can put this in official release

Reply:
Me neither. Changed to some mocked name.